packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -0,0 +1,64 @@
+import { attr } from '@microsoft/fast-element';
+import { DesignSystem, FoundationElement } from '@microsoft/fast-foundation';
+import { template } from '../theme-provider/template';
+import { styles } from './styles';
+import {
+    WaferMapColorBy,
+    WaferMapColorsScale,
+    WaferMapDataType,
+    WaferMapDie,
+    WaferMapOrientation,
+    WaferMapQuadrant
+} from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-wafer-map': WaferMap;
+    }
+}
+
+/**
+ * A nimble-styled WaferMap
+ */
+export class WaferMap extends FoundationElement {
+    @attr({
+        attribute: 'wafer-quadrant'
+    })
+    public waferQuadrant!: WaferMapQuadrant;
+
+    @attr({
+        attribute: 'wafer-orientation'
+    })
+    public waferOrientation!: WaferMapOrientation;
+
+    @attr({
+        attribute: 'max-characters'
+    })
+    public maxCharacters!: number;
+
+    @attr({
+        attribute: 'wafer-datatype'
+    })
+    public waferDataType!: WaferMapDataType;
+
+    @attr({
+        attribute: 'colorBy'
+    })
+    public colorBy: WaferMapColorBy = WaferMapColorBy.hardBin;
+
+    @attr({
+        attribute: 'highlighted-values'
+    })
+    public highlightedValues!: string[];
+
+    public dice!: WaferMapDie[];
```

Comment:
Is "dice" semantically correct for multiple dies in the industry? I tend to think that "dice" is referring specifically to multiple game dies (though now it is also used for the singular cube). I would have expected "dies" for this (see entry 3 [here](https://www.merriam-webster.com/dictionary/die#dictionary-entry-2)).

Reply:
Updated to "dies", thank you. :)
I changed before into the "dice" because this was the word used in the old code, but we search also on this topic and the "dies" is the correct name. See https://en.wikipedia.org/wiki/Wafer_(electronics) and also https://www.artwork.com/package/wmapconvert/manual_v2/glossary_of_terms.html